### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by 3kori**
I hope this PR is welcomed. I have been practicing git commands and this repo has been really helpful. Thank you.

### CODE REVIEW

File: content/docs/project/file/contents.lr [hunk 1]
```diff
@@ -206,7 +206,7 @@ locale = fr
 ```
 
 ### `[attachment_types]`
-Lektor does some basic attachment type detection based on file extension. This is what powers the `this.attachemnts.images` and `this.attachments.videos` attributes for instance. If the built-in map does not cover your file extension you can extend it or add new attachement types on a project by project basis.
+Lektor does some basic attachment type detection based on file extension. This is what powers the `this.attachements.images` and `this.attachments.videos` attributes for instance. If the built-in map does not cover your file extension you can extend it or add new attachment types on a project by project basis.
```
Comment (dairiki) [inline content/docs/project/file/contents.lr:209]: Should be `this.attachments.images`

**Review by dairiki — CHANGES_REQUESTED** [review]
@3kori The PR is most welcomed. Thank you!

I did, however, notice a couple of misspellings in your corrections of previous misspellings.

**Comment by 3kori** [comment]
I am sorry for making a mistake.

**Comment by dairiki** [comment]
@3kori No problem!  Thank you for fixing all the typos you've found!

**Pull request merged**